src/js/smooch.jsx

```diff
@@ -25,14 +25,12 @@ import { waitForPage } from 'utils/dom';
 function renderWidget() {
     const el = document.createElement('div');
     el.setAttribute('id', 'sk-holder');
-    el.className = 'sk-noanimation';
```

Comment:
Before we were applying `sk-close` on the initial render to put it at the bottom, but `sk-close` has an animation.. that was causing the same glitch that was reported. Adding `sk-animation` until it was fully rendered kinda worked around that problem. Now, the initial state of the widget is neither opened or closed, so we don't apply `sk-close` or `sk-appear` and the initial state of the widget is to be at the bottom. So this class is no longer needed.